ui/components/ui/currency-display/currency-display.component.js

```diff
@@ -53,7 +53,7 @@ CurrencyDisplay.propTypes = {
   'className': PropTypes.string,
   'currency': PropTypes.string,
   'data-testid': PropTypes.string,
-  'denomination': PropTypes.oneOf([GWEI]),
+  'denomination': PropTypes.oneOf([GWEI, ETH]),
```

Comment:
For some reason I was finding after adding max fee per gas that I was getting an error about ETH not being a valid denomination. Adding ETH to the list fixed this issue, but I am curious whether there is a better way to solve this problem as I do not totally understand how this works.

Reply:
Yeah that's very odd given that there are other UserPreferencedCurrencyDisplay instances getting passed `ETH` as denomination in the same component...

Reply:
That being said this should be included here. So I guess propTypes is as unreliable as we've always assumed

Reply:
Indeed :)